src/utils/validateData.ts

```diff
@@ -0,0 +1,37 @@
+import validator from "validator";
+
+interface ValidateError {
+    valid: Boolean;
+    description: string;
+};
+
+export const validateData = (
+    email: string, 
```

Comment:
This is really not looking good with chaining ifs, but i think in TS its a best you can do ....